### DISCUSSION TITLE
[Kavita] Have Seperate Icon for Online Users

### DISCUSSION THREAD

**Discussion by impishlucy** [Kareadita/Kavita#3283] [Ideas]
### Idea Description

The active online user count is merged with the task symbol, this is unintuative and confused me multiple times when i looked what task could be running only to see "no current tasks".
![image](https://github.com/user-attachments/assets/c899b549-12fb-4191-9886-5b23582a31e7)

What did you expect?
The active user count should be a seperate Icon to avoid confusion
![image](https://github.com/user-attachments/assets/3b61bfe9-d00d-45ff-93cf-c5357442073e)

### Idea Category

Web UI

### Duration of Using Kavita

3+ Years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
This adds more visual clutter. The way it is currently handled is the same way that plex handles it.
